src/utils/discord/getUsername.ts

```diff
@@ -0,0 +1,2 @@
+import { IDiscordUser } from "./interfaces/user"
```

Comment:
useless file?

since u moved it to https://github.com/mirage-software/Kabooty/blob/112781b5af8d8e232da95960ab1b26807cb420b5/src/stores/discord.ts